geppetto-showcase/package.json

```diff
@@ -5,8 +5,6 @@
   "repository": "https://github.com/MetaCell/geppetto-website.git",
   "license": "MIT",
   "scripts": {
-    "prepare": "yarn link:yalc",
```

Comment:
Those lines were here to be used to setup the local dev version of the projects that depends on geppetto, they are not installed/used otherwise by npm, they are not necessary anymore with the PR update?